### ISSUE TITLE
Getting addon errors on load with the plugin

### ISSUE THREAD

**Issue by AManNamed2** [Tanza3D/discord-no-mosaic#5]
![image](https://user-images.githubusercontent.com/50536667/222936199-31168e50-b7d8-4f03-bebd-91b171631d0b.png)

**Comment by Tanza3D** [comment]
hm, do you possibly have the friends list or such open behind the settings window instead of a chat? regardless, should fix it, but if you make sure to go to a chat window first it may work

**Comment by Tanza3D** [comment]
see if that fixes it

**Comment by AManNamed2** [comment]
Yup that worked, preview quality seems to get nuked now but you win some you loose some

**Comment by Tanza3D** [comment]
nuked as in it's way too high quality? or low

**Comment by Tanza3D** [comment]
because as of right now it's loading the images with no compression or anything, i have to find a way around that but it should look even crisper than ever... also take longer to load than ever but
